### ISSUE TITLE
Add is_a link between mitochondrial tRNA modification and mitochondrial tRNA processing

### ISSUE THREAD

**Issue by pfey03** [geneontology/go-ontology#12757]
GO:0070900: mitochondrial tRNA modification should be a is_a child of GO:0090646: mitochondrial tRNA processing.
. . <-  GO:0090646: mitochondrial tRNA processing
. . . <- is_a GO:0070900: mitochondrial tRNA modification
I find the separation between 'mitochondrial tRNA modification' and 'mitochondrial tRNA processing' very small to begin with. But it's strange that these two terms are not even related, because further up 'tRNA modification' is a child of 'tRNA processing'. 
Thanks,
petra

**Comment by paolaroncaglia** [comment]
Hi @pfey03 ,
FYI, I haven’t had a chance to work on this ticket until now, and I’ll be on leave next week. Thanks for your patience.

**Comment by pfey03** [comment]
Thanks Paola, have a great week end and next week!

**Comment by ValWood** [comment]
Isn't the problem here the fact that tRNA modification is a child of tRNA processing?
Processing refers to alterations to the primary transcript doesn't it (nucleotide cleavage or addition), but excludes modifications? Is that correct?

**Comment by paolaroncaglia** [comment]
@pfey03 , @ValWood , @hdrabkin

Welcome to the daily can of worms.
Looks like there’s a wider problem with consistency in the use of ‘modification’ and ‘processing’ in GO. Probably because the usage in the literature isn’t. Looking at different macromolecule branches:

protein metabolic process
- (is_a) protein maturation
- - (is_a) protein processing
- (is_a) protein modification process
=> ‘modification’ and ‘processing’ are distinct; there is the presence of an intermediate parent (’maturation’) above ‘processing’

DNA metabolic process
- (is_a) DNA modification

RNA metabolic process
- (is_a) RNA modification
- (is_a) RNA processing

ncRNA metabolic process
- (is_a) ncRNA processing
- - (is_a) tRNA processing
- - - (is_a) tRNA modification
(same for rRNA processing and others)

macromolecule modification = The covalent alteration of one or more monomeric units in a polypeptide, polynucleotide, polysaccharide, or other biological macromolecule, resulting in a change in its properties.

I’ll ask @hdrabkin for help here. The situation may be different for different types of macromolecules, but we need to be consistent. What would be the correct way to go? Thanks.

**Comment by krchristie** [comment]
A long time ago, when @hdrabkin, Ceri van Slyke and I worked on some rRNA processing and ribosome biogenesis stuff, we felt that "modification" was included in "processing" for rRNA, as one of the many types of processing that can occur.

Harold may be more up-to-date on this. He's on vacation this week and then will go to a PRO meeting next week, so he probably won't respond till after that.

**Comment by paolaroncaglia** [comment]
Thanks @krchristie .

@pfey03 and @ValWood , I'll set this to pending then, waiting for @hdrabkin to take a look.
Feel free to 'ping' him or me by commenting here if too long goes by without an update. Thanks.

**Comment by cmungall** [comment]
> Welcome to the daily can of worms.

This is a great phrase

**Comment by paolaroncaglia** [comment]
Hi @hdrabkin ,
When you have a chance, could you please take a look at this ticket, and summarise what the best action plan would be. Many thanks.

**Comment by paolaroncaglia** [comment]
Hi @pfey03,

Going back to your initial request in this ticket:

“GO:0070900: mitochondrial tRNA modification should be a is_a child of GO:0090646: mitochondrial tRNA processing.
. . <- GO:0090646: mitochondrial tRNA processing
. . . <- is_a GO:0070900: mitochondrial tRNA modification
I find the separation between 'mitochondrial tRNA modification' and 'mitochondrial tRNA processing' very small to begin with. But it's strange that these two terms are not even related, because further up 'tRNA modification' is a child of 'tRNA processing’.”

My understanding is that you came across this while you were annotating to (or looking for) related terms, but that you do not strictly need this to be resolved in order to be able to annotate papers you’re working on. Could you please confirm if this is correct?

Many thanks.

**Comment by paolaroncaglia** [comment]
Looking at this ticket again. In summary:
The use of ‘modification’ and ‘processing’ in class labels in GO doesn’t seem to be consistent (see https://github.com/geneontology/go-ontology/issues/12757#issuecomment-259110495). Is this because the usage in the literature is indeed different depending on the type of macromolecule, and if so, do we need to add definition comments to explain any discrepancy? Or is the inconsistency a problem we need to fix more radically, and if so what branch(es) should we rename/restructure, and how?
The impression was that @hdrabkin may be the best person to lead on this. I’ll assign to him. (My understanding is that this issue is not preventing any specific annotation, if that helps prioritising.) Thanks.

**Comment by pfey03** [comment]
@paolaroncaglia to answer your question from December (sorry went on break mid December and then fell through cracks). 
I saw this because I annotated a paper/gene, but found GO:0070900: mitochondrial tRNA modification a good term as reflected in the definition, for what I needed (PMID:21307182, TLP4 / tip4). 
Since the same paper had functional data for a non mitochondrial protein, I saw the discrepancy between the 'tRNA modification/processing' relationship versus the 'mitochondrial tRNA modification /processing' NON relationship. 
In sum, I could use the term I needed to annotate and just thought the structural inconsistency has to be addressed. That it's such a huge can of worms, really sorry for opening!!
